pkg/pipelinerunmetrics/metrics_test.go

```diff
@@ -375,8 +375,87 @@ func TestRecordRunningPipelineRunsCount(t *testing.T) {
 	metricstest.CheckLastValueData(t, "running_pipelineruns_count", map[string]string{}, 1)
 }
 
+func TestRecordRunningPipelineRunsResolutionWaitCounts(t *testing.T) {
+	for _, tc := range []struct {
+		status      corev1.ConditionStatus
+		reason      string
+		prWaitCount float64
+		trWaitCount float64
```

Comment:
can you add a test case for multiple taskruns/pipelineruns?

Reply:
will do, though as I type today is EOB Friday, so it will be early next week hopefully.